### ISSUE TITLE
Setting to disable Amazon SQS

### ISSUE THREAD

**Issue by galphanet** [kryptco/kr#180]
We would like an option to disable SQS and only use Bluetooth (or USB as #139) communication between the phone and the computer.

The reason is we think that to improve security, the phone should be in range of the computer.
Imagine if you "Alway allow" access to the key and the computer is stolen, the phone can still be used to sign access.

**Comment by agrinman** [comment]
Hi @galphanet, we do plan to support a bluetooth only policy for our teams product (coming soon) -- but to your point I agree it makes sense to add to Krypton Core itself.

**Comment by galphanet** [comment]
Thanks for your fast reply.
Would it be possible to have an estimated time line on your website?

We would be interested by your team product.

**Comment by agrinman** [comment]
Absolutely, please send me an email to “alex at krypt.co” and I can fill you in on the details!
